apps/site/i18n.tsx

```diff
@@ -1,22 +1,28 @@
 import { importLocale } from '@node-core/website-i18n';
+import deepmerge from 'deepmerge';
```

Comment:
Is a new dependency really needed here? 🤔

Something like this:

```js
function deepMerge(a, b) {
  var o = {}

  for (let key in a) {
    let originalValue = a[key],
        value = originalValue

    if (b.hasOwnProperty(key)) {
      value = b[key]
    }

    if (typeof value === 'object') {
      value = deepMerge(originalValue, value)
    }

    o[key] = value
  }

  return o
}
```

(I bet it can be simplified even further)

Reply:
Sure, sounds good! `deepmerge` can in fact also merge arrays, I guess it's a good call to go for something simpler.